ugvc/utils/misc_utils.py

```diff
@@ -354,60 +271,3 @@ def filter_valid_queries(df_test: pd.DataFrame, queries: dict, verbose: bool = F
                 logger.warning(f"Filter query {filter_query} caused an exception, skipping.")
 
     return valid_filters
-
-
-def read_hdf(
```

Comment:
was moved to metrics_utils:
https://github.com/Ultimagen/ugbio-utils/blob/7cd16e1ca18c33e3a1c65d6f841dbbbd31b9bd63/src/core/ugbio_core/metrics_utils.py#L90